### ISSUE TITLE
qt-gui: silently ignoring errors during open

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#204]
When kdbGet() returns -1 during starting qt-gui the error is only printed to stdout, but the user is not informed and only sees empty hierarchies.

Steps to reproduce, e.g.
```
# kdb mount /etc/security/limits.conf system/limits augeas lens=Limits.lns
# sudo chmod 600  /etc/security/limits.conf
$ bin/qt-gui
Error (#9) occurred!
Description: Insufficient permissions to open configuration file for writing. You might want to retry as root.
Ingroup: kdb
Module: 
At: /home/markus/Projekte/Elektra/current/src/plugins/augeas/augeas.c:378
Reason: Keine Berechtigung
Mountpoint: system/limits
Configfile: /etc/security/limits.conf
```

**Comment by markus2330** [comment]
not yet confirmed by testcase

**Comment by 0003088** [comment]
It worked when I set *isError=true*. Could you share your testcase please?

**Comment by markus2330** [comment]
I just reopend because I did not have time to execute the testcase when I merged 0.0.7. Now I did it and it worked flawlessly ;) Nicely done, thank you!

What do you mean with *isError=true*, I could not find that text.

**Comment by 0003088** [comment]
Nevermind my comment. I was referring to a line of code in main.cpp. I tested the fix by setting the variable *isError* to true (and by using your testcase from your initial posting) and regarded it as working. Good to know you came to the same conclusion. I am closing this issue now.
